server/routes/index.js

```diff
@@ -0,0 +1,27 @@
+import client from '../helpers/redis';
+
+import daysBetween from '../helpers/daysBetween';
+
+export default (app) => {
+  app.get('/worker-status', (req, res) => {
+
+    return client.mget('startTime', 'message', 'jobDone', (err, response) => {
+      if (err) {
+        return res.status(200).send({
```

Comment:
I believe a status of `400` is more suitable for an error response.

Reply:
Thanks for spotting it, I totally missed it @IAMOTZ